test/Removal.t.sol

```diff
@@ -94,6 +94,60 @@ contract Removal_mintBatch_reverts_mint_to_wrong_address is UpgradeableMarket {
   }
 }
 
+contract Removal_mintBatch_zero_amount_removal is UpgradeableMarket {
+  function test() external {
+    DecodedRemovalIdV0[] memory ids = new DecodedRemovalIdV0[](1);
+    ids[0] = DecodedRemovalIdV0({
+      idVersion: 0,
+      methodology: 1,
+      methodologyVersion: 0,
+      vintage: 2018,
+      country: "US",
+      subdivision: "IA",
+      supplierAddress: _namedAccounts.supplier,
+      subIdentifier: _REMOVAL_FIXTURES[0].subIdentifier + 1
```

Comment:
The `+ 1` isnt needed here

Reply:
ah right that was from pulling from a different test that was trying to mint a second removal and keep it unique